aws/sdk/build.gradle.kts

```diff
@@ -72,6 +72,8 @@ fun eventStreamAllowList(): Set<String> {
 }
 
 fun generateSmithyBuild(services: AwsServices): String {
+    val awsConfigVersion = properties.get("smithy.rs.runtime.crate.version")
```

Comment:
This PR makes all the runtime crates use the same version numbers. They all use `smithy.rs.runtime.crate.version` now. This will result in a version number jump from ~0.15.0 to ~0.46.0 for the AWS runtime crates on the first release that has these changes, but that should be fine.